extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -817,8 +817,9 @@ public function changePassword() {
 				}
 
 				$user->setPassword( $newPassword );
+                $user->invalidateToken();
```

Comment:
I wonder if not better if we moved this `invalidateToken()` into `setPassword()`? You would need to check when it is called and if this would be a correct behaviour.